### ISSUE TITLE
[Suggestion] Config for the harvesting tweak

### ISSUE THREAD

**Issue by Lordician** [alcatrazEscapee/no-tree-punching#43]
I know that the mechanic is quite integral to how the mod works, but i am playing with some people that do not quite agree with the harvesting change and the requirements it brings up due to it.
I tried putting in *:* as the only entry at the breakable list, but that did not work sadly.
(I expected/hoped it to just take any modid : any registery name)

So a config to disable the harvest mechanic would be nice or a way to just add everything to the list by means of asterisks.
(any modid = *:registery_name, any block from certain mod= modid:* and any block from any mod: *:*)

I know it is a bit specific and it kind of makes some things redundant, but i don't want to remove the entire mod because it still has a lot of neat things that work without the harvesting tweak.

**Comment by alcatrazEscapee** [comment]
So I am open to adding this, but I would reiterate (as you have already said) that it is quite integral to the mod. However, due to the fact that it is technically already possible I will add this.

I am interested, what kind of requirements are you speaking of? If there are specific concerns I would like to know, to try and minimize them some other game play related way.

**Comment by Lordician** [comment]
Harvesting requires the right tool. Making an axe in the mod is first done by making a flint axe by default or by enabling the vanilla recipes again, having planks and sticks.
While i reason that flint is pretty easy to come by at the start, i am working with people who are... unconvinced.
And you cannot get planks unless you cut wood, which you cannot punch.

Tl;DR i am trying to keep most of this mod in while having to butcher vanilla tweaks due to my friends being hung up to vanilla ways. I cannot convince them otherwise and i don't want to get rid of this mod but also not play without them.

I thank you for wanting to add the feature. Makes it a lot easier for me to not have to lose this mod completely.

**Comment by alcatrazEscapee** [comment]
So you are trying to use this mod without any of the vanilla changes. Well that is an interesting use case.

**Comment by alcatrazEscapee** [comment]
"notreepunching:\*" -> Any block from mod id "notreepunching"
"\*:stone" -> any mod id with registry path "stone" (?)
"\*:\*" -> probably the one you were looking for
